docs/dm/assets.md

```diff
@@ -30,6 +30,14 @@ Once uploaded, your files will be accessible in-game.
 
 When multiple files are selected a progress bar will be visible to update you on the state.
 
+### Downloading Files
+
+To download files, select the asset(s) you want to download and click the download files icon.
+This will open a download dialogue, providing you with a single `\*.paa` file (the extension meaning **p**lanar **a**lly **a**sset).
```

Comment:
```suggestion
This will open a download dialogue, providing you with a single `*.paa` file (the extension meaning **p**lanar **a**lly **a**sset).
```